jira7/jira_connector.bal

```diff
@@ -123,9 +123,9 @@ public function JiraConnector::getAllProjectSummaries() returns ProjectSummary[]
 
     endpoint http:Client jiraHttpClientEP = self.jiraHttpClient;
     ProjectSummary[] projects = [];
-    http:Request request = new;
+    http:Request outRequest = new;
 
-    var httpResponseOut = jiraHttpClientEP->get("/project?expand=description", request);
+    var httpResponseOut = jiraHttpClientEP->get("/project?expand=description", request=outRequest);
```

Comment:
Formatting issue here for not having spaces. 
Can we please change this to **iraHttpClientEP -> get** ?

Reply:
Fixed